kong/reports.lua

```diff
@@ -438,4 +454,6 @@ return {
   end,
   send = send_report,
   retrieve_redis_version = retrieve_redis_version,
+  -- exposed for unit test
+  create_counter = create_counter,
```

Comment:
not a blocker: in `kong.runloop.balancer` we exposed private functions for testing using underscore-prefixed names (`_create_counter`)

Reply:
changed, thanks!